release/stable/kali-rolling/docker/Dockerfile

```diff
@@ -0,0 +1,80 @@
+# Docker image file that describes an Kali Rolling image with PowerShell installed from tar.gz
+# Implemented in two stages, one that downloads and uncompresses the tar.gz
+# The next stage simply copies the files from the first stage, 
+# effectively removing the tar.gz and packages that are not needed from the final image.
+
+# Define arg(s) needed for the From statement
+ARG fromTag=kali-rolling
+ARG imageRepo=kalilinux/kali-linux-docker
+
+FROM ${imageRepo}:${fromTag} AS installer-env
+
+# Define Args for the needed to add the package
+ARG PS_VERSION=6.1.0
+ARG PS_PACKAGE=powershell_${PS_VERSION}-1.debian.9_amd64.deb
+ARG PS_PACKAGE_URL=https://github.com/PowerShell/PowerShell/releases/download/v${PS_VERSION}/${PS_PACKAGE}
+ARG PS_INSTALL_VERSION=6
+
+# Download the Linux tar.gz and save it
+ADD ${PS_PACKAGE_URL} /tmp/powershell.deb
+
+# Install dependencies
+RUN wget \
```

Comment:
Can you replace the wget statements with docker `ADD` commands.

Reply:
This isn't blocking either.

Reply:
- Replaced `wget` linux bash command with `ADD` Dockerfile command